src/sth_database.js

```diff
@@ -300,9 +306,11 @@
    *  if it does not exist
    * @param {boolean} shouldStoreHash Flag indicating if the collection hash should be stored in case the collection
    *  is created
+   * @param {boolean} shouldTruncate Flag indicating if the collection should be truncate according to the requested data
```

Comment:
Fixed in https://github.com/telefonicaid/IoT-STH/commit/b36a70fdb47783cd5022406b65d6dcd1c1a3a60a